compiler/passes/scopeResolve.cpp

```diff
@@ -425,6 +425,23 @@ static void scopeResolve(ForallStmt*         forall,
   scopeResolve(loopBody->body, bodyScope);
 }
 
+static void scopeResolveForeachLoop(ForLoop* foreach,
+                                    const ResolveScope* parent)
+{
+  INT_ASSERT(foreach->isOrderIndependent());
+
+  BlockStmt*    loopBody     = foreach->loopBody();
+  ResolveScope* bodyScope = new ResolveScope(loopBody, parent);
```

Comment:
Weird whitespaces probably due to changes after a copy/paste.

(I am not a fan of creating tables with whitespaces but we have that a lot in the compiler. My preference would be to avoid it. If you do it, the line above should be adjusted)